libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1267,7 +1267,11 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_variant",
-            "values": {"c++17": 202102},
+            "values": {
+                "c++17": 202102,  # std::visit for classes derived from std::variant
+                # "c++20": 202106,  # Fully constexpr std::variant
+                # "c++26": 202306,  # Member visit
```

Comment:
```suggestion
                # "c++26": 202306,  # Member visit (implemented)
```

Reply:
I believe this FTM can be bumped now. I'll do the cleanup.